QEfficient/generation/text_generation_inference.py

```diff
@@ -144,15 +144,17 @@ def cloud_ai_100_exec_kv_helper(
     enable_debug_logs: bool = False,
     stream: bool = True,
     write_io_dir: Optional[str] = None,
+    session=None,
```

Comment:
`input_len` variable is not being used anywhere.

Reply:
Updated